### PR TITLE
Assignment 3 - Bryan Caesar

### PR DESCRIPTION
**Pull request by bcaesar42**

### CODE REVIEW

**Comment by bcaesar42** [comment]
Not finished yet.

File: SecretSanta/src/SecretSanta.Api/Controllers/UserController.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            if (user is null)
+            {
+                return BadRequest();
+            }
+
+            _UserService.AddUser(DTO.User.ToEntity(user));
+            return Ok();
+        }
+    }
+}
```
Comment (Keboo) [inline SecretSanta/src/SecretSanta.Api/Controllers/UserController.cs]: The assignment called for the controllers to expose "Create/Update/Delete users". I only see one of those here.

File: SecretSanta/test/SecretSanta.Api.Tests/ValuesControllerTests.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+using System.Collections.Generic;
+using System.Text;
+
+namespace SecretSanta.Api.Tests
+{
+    [TestClass]
+    public class ValuesControllerTests
+    {
+        
+    }
```
Comment (Keboo) [inline SecretSanta/test/SecretSanta.Api.Tests/ValuesControllerTests.cs]: You can omit this class.

File: SecretSanta/src/SecretSanta.Api/Controllers/GroupController.cs [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    {
+        private readonly IGroupService _GroupService;
+
+        public GroupController(IGroupService groupService)
+        {
+            _GroupService = groupService ?? throw new ArgumentNullException(nameof(groupService));
+        }
+
+        [HttpGet]
+        public ActionResult<List<DTO.Group>> FetchAll()
```
Comment (Keboo) [inline SecretSanta/src/SecretSanta.Api/Controllers/GroupController.cs]: This method is not covered by any unit tests.

File: SecretSanta/src/SecretSanta.Api/Controllers/GroupController.cs [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+            {
+                return BadRequest();
+            }
+
+            _GroupService.UpdateGroup(DTO.Group.ToEntity(group));
+            return Ok();
+        }
+
+        [HttpPost]
+        public ActionResult<List<DTO.User>> GetUsers(int groupId)
```
Comment (Keboo) [inline SecretSanta/src/SecretSanta.Api/Controllers/GroupController.cs]: This method is not covered by any unit tests.

File: SecretSanta/src/SecretSanta.Api/Controllers/UserController.cs [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using SecretSanta.Domain.Services;
+
+namespace SecretSanta.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class UserController : ControllerBase
```
Comment (Keboo) [inline SecretSanta/src/SecretSanta.Api/Controllers/UserController.cs]: It appears there are no unit tests for this class.

**Pull request closed without merging**